### DISCUSSION TITLE
Option to disable Plans, Donations and multiple inventory locations.

### DISCUSSION THREAD

**Discussion by mikesnoeren** [craftcms/commerce#3485] [Ideas] [answered]
For a lot of webshops, the options above are not required. Having them is great, but they clutter up the CMS a bit. It would be nice if we could disable those.

**Comment by nfourtythree** [comment, marked as answer]
Hi @mikesnoeren 

Thank you for your feedback. Adding some form of configuration or settings for these is something we can consider in the future.

For the moment, you can achieve this with a small piece of code in a custom module. 

```php
use craft\base\Event;
use craft\events\RegisterCpNavItemsEvent;
use craft\web\twig\variables\Cp;

// ...

Event::on(Cp::class, Cp::EVENT_REGISTER_CP_NAV_ITEMS, function(RegisterCpNavItemsEvent $event) {
    foreach ($event->navItems as &$navItem) {
        if ($navItem['url'] !== 'commerce') {
            continue;
        }

        foreach (['donations', 'subscription-plans'] as $key) {
            unset($navItem['subnav'][$key]);
        }
    }
});
```

Hope this helps, thanks!
